lib/src/components/viewOptions/ViewOptions.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react';
+import { ButtonIcon, ButtonTertiary } from 'lib';
+import { SearchInput } from '../../modules/searchInput/SearchInput';
+import Dropdown from '../../../Dropdown';
+import Icon from '../../../Icon';
+
+export function ViewOptions({
+  onChangeSearchHandler,
+  onClearSearchHandler,
+  searchValue,
+  onViewChange,
+  onSortChange,
+  sortingValues,
+  actions
+}) {
+  const [view, setView] = useState('grid');
```

Comment:
💚 It could be useful to attach the view types to the component, for use in the app
```javascript
ViewOptions.viewTypes = {
  grid: 'grid',
  list: 'list'
}
```
that way I don't have to open up the component to know what the view types are when I am implementing list and gird views in the app.